tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -359,6 +359,66 @@ impl<OpOrder: PBSOrderMarker> From<CompressedCiphertextBase<OpOrder>> for Cipher
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct CompactPublicCiphertextList<OpOrder: PBSOrderMarker> {
```

Comment:
For the naming Public here is not required I believe ? same thing elsewhere for integers

Reply:
I choose that naming as these ciphertext can only be created from a CompactPublicKey but we can change that

Reply:
understood, we currently don't have the information about where a CT comes from (secret/public key) I don't think the distinction is necessary

Reply:
So we remove it ?

Reply:
yeah Public Here is not necessary IMO